### DISCUSSION TITLE
problem with rpi3 boot

### DISCUSSION THREAD

**Discussion by m-gavrilyuk** [ltsp/ltsp#242] [Q&A]
I use how-to https://ltsp.org/docs/installation/raspbian/ 

![IMG_20200203_222539](https://user-images.githubusercontent.com/60624717/73683960-4e5dc280-46d4-11ea-9dd5-20a883942b1a.jpg)
I tried to DEBUG_SHELL=1 but it's did't work 
Please help!

**Comment by alkisg** [comment]
Hi, please read the note at the top of that page; Raspbian support in LTSP currently needs remote assistance via VNC. Users interested in this can mail me at alkisg at gmail.
After some funding is found, it'll be fully automated.
The instructions to netboot a single client in NFS_RW mode do work fine though.
